Description: Upstream changes introduced in version 3.2-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 canaima-base (3.2-2) unstable; urgency=low
 .
   * [75bf194] Quitando escritorios inexistentes del Recommends en
     debian/control
   * [6a4a3a2] Nueva versión
   * [0eccf1a] Modificaciones de estilo
 .
 The person named in the Author field signed this changelog entry.
Author: Luis Alejandro Martínez Faneyth <martinez.faneyth@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- canaima-base-3.2.orig/COPYING
+++ canaima-base-3.2/COPYING
@@ -1,9 +1,7 @@
 Derechos de autor:
 
-    Copyright (C) 2010 Luis Alejandro Martínez Faneyth
-    Copyright (C) 2010 Diego Alberto Aguilera Zambrano
-    Copyright (C) 2010 Carlos Alejandro Guerrero Mora
-    Copyright (C) 2010 Francisco Javier Vásquez Guerrero
+    Copyright (C) 2010 - 2012 Luis Alejandro Martínez Faneyth
+    Copyright (C) 2010 - 2012 Francisco Javier Vásquez Guerrero
 
 Licencia:
 
--- canaima-base-3.2.orig/AUTHORS
+++ canaima-base-3.2/AUTHORS
@@ -1,6 +1,4 @@
-Autores:
+Mantenedor Actual:
 
- * Luis Alejandro Martínez Faneyth <martinez.faneyth@gmail.com>
- * Diego Alberto Aguilera Zambrano <daguilera85@gmail.com>
- * Carlos Alejandro Guerrero Mora <guerrerocarlos@gmail.com>
+ * Luis Alejandro Martínez Faneyth <luis@huntingbears.com.ve>
  * Francisco Javier Vásquez Guerrero <franjvasquezg@gmail.com>
--- canaima-base-3.2.orig/THANKS
+++ canaima-base-3.2/THANKS
@@ -0,0 +1,9 @@
+
+ * Diego Alberto Aguilera Zambrano <daguilera85@gmail.com>
+ * Carlos Alejandro Guerrero Mora <guerrerocarlos@gmail.com>
+ * Francisco Javier Vásquez Guerrero <franjvasquezg@gmail.com>
+ * José Miguel Parrella Romero <bureado@debian.org>
+ * Carlos David Marrero <cdmarrero2040@gmail.com>
+ * Orlando Andrés Fiol Carballo <ofiol@indesoft.org.ve>
+ * Jesús Ignacio Lara Giménez <jesuslara@devel.com.ve>
+
--- canaima-base-3.2.orig/TODO
+++ canaima-base-3.2/TODO
@@ -1,15 +1,2 @@
-ANOTACIONES MISCELÁNEAS
 
-* 
-
-GRANDES MODIFICACIONES
-
-* 
-
-PEQUEÑAS MODIFICACIONES
-
-* Now running lintian...
-W: canaima-escritorio-gnome source: non-native-package-with-native-version
-E: canaima-escritorio-gnome: copyright-should-refer-to-common-license-file-for-gpl
-W: canaima-escritorio-gnome: debian-changelog-line-too-long line 1
 
--- /dev/null
+++ canaima-base-3.2/Makefile
@@ -0,0 +1,37 @@
+# Makefile
+
+SHELL := sh -e
+
+SCRIPTS = debian/postinst debian/postrm
+
+all: test build
+
+test:
+
+	@printf "Comprobando sintaxis de los scripts de shell ["
+	@for SCRIPT in $(SCRIPTS); do \
+		sh -n $${SCRIPT}; \
+		checkbashisms -f -x $${SCRIPT} || true; \
+		printf "."; \
+	done
+	@printf "]\n"
+
+build:
+
+	@echo "Nada para compilar!"
+
+install:
+
+	@mkdir -p $(DESTDIR)/usr/share/canaima-base
+	@mkdir -p $(DESTDIR)/etc/canaima-base/alternatives
+	@cp -r alternatives $(DESTDIR)/usr/share/canaima-base/
+
+uninstall:
+
+	@rm -rf $(DESTDIR)/usr/share/canaima-base
+
+clean:
+
+distclean:
+
+reinstall: uninstall install
--- /dev/null
+++ canaima-base-3.2/alternatives/locale.gen
@@ -0,0 +1 @@
+es_VE.UTF-8 UTF-8
--- /dev/null
+++ canaima-base-3.2/alternatives/sources.list
@@ -0,0 +1,20 @@
+#
+# Repositorios de Canaima GNU/Linux
+#
+
+# Repositorio Antiguo (3.0)
+#deb http://paquetes.canaima.softwarelibre.gob.ve/ roraima main aportes no-libres
+
+# Repositorio roraima VIT
+deb http://vit.canaima.softwarelibre.gob.ve/repositorio/ roraima usuarios
+
+# Repositorio Estable (3.1)
+deb http://paquetes.canaima.softwarelibre.gob.ve/ auyantepui main aportes no-libres
+#deb-src http://paquetes.canaima.softwarelibre.gob.ve/ auyantepui main aportes no-libres
+
+# Repositorio de Desarrollo (4.0)
+#deb http://paquetes.canaima.softwarelibre.gob.ve/ kukenan main aportes no-libres
+#deb-src http://paquetes.canaima.softwarelibre.gob.ve/ kukenan main aportes no-libres
+
+
+
--- /dev/null
+++ canaima-base-3.2/alternatives/locale
@@ -0,0 +1,3 @@
+LANG="es_VE.UTF-8"
+LANGUAGE="es_VE:es"
+LC_ALL="es_VE.UTF-8"
--- /dev/null
+++ canaima-base-3.2/alternatives/keyboard
@@ -0,0 +1,16 @@
+# Check /usr/share/doc/keyboard-configuration/README.Debian for
+# documentation on what to do after having modified this file.
+
+# The following variables describe your keyboard and can have the same
+# values as the XkbModel, XkbLayout, XkbVariant and XkbOptions options
+# in /etc/X11/xorg.conf.
+
+XKBMODEL='pc105'
+XKBLAYOUT='latam'
+XKBVARIANT=""
+XKBOPTIONS=""
+
+# If you don't want to use the XKB layout on the console, you can
+# specify an alternative keymap.  Make sure it will be accessible
+# before /usr is mounted.
+# KMAP=/etc/console-setup/defkeymap.kmap.gz                                                  
